rtl/include/eclhelper.hpp

```diff
@@ -1683,10 +1677,6 @@ struct IHThorSortArg : public IHThorArg
     virtual bool hasManyRecords() = 0;
     virtual double getTargetSkew() = 0;
     virtual ICompare * queryCompareSerializedRow()=0;                           // null if row already serialized, or if compare not available
-};
-
-struct IHThorAlgorithm : public IInterface
-{
     virtual unsigned getAlgorithmFlags() = 0;
```

Comment:
I am very tempted to move both of these functions into IHThorArg since they should be applicable to any activity.

Also, if the algorithm is only allowed to be a constant then the name could be stored as a constant inn the graph instead.

Reply:
I thought we had decided to deprecate algorithm in favour of the hint mechanism...

Reply:
It looks like it is.  Maybe this should be deleted completely then - and force that algorithm is constant.